src/ray/protobuf/usage.proto

```diff
@@ -99,4 +99,9 @@ enum TagKey {
   // Logical operators, stored in JSON format with operator name and count.
   // Example: {"MapBatches": 2, "Filter": 1}
   DATA_LOGICAL_OPS = 400;
+
+  // AIR
+  // AIR trainer, or custom trainer that instantiates BaseTrainer interface.
+  // Example: TorchTrainer
+  AIR_TRAINER = 401;
```

Comment:
Should we take another prefix, e.g. 5? 
```suggestion
  AIR_TRAINER = 500;
```

Reply:
Also I am conflicted if we should name it `AIR_TRAINER` or `TRAIN_TRAINER` 😂 

@gjoliver?

Reply:
what the ... let's just standardize on AIR Trainer?

Reply:
Yea, lets reserve another chunk of indexes. 4xx is for data.